cognito-post-authentication/src/test/java/no/unit/nva/cognito/model/EventTest.java

```diff
@@ -0,0 +1,22 @@
+package no.unit.nva.cognito.model;
+
+import java.io.File;
+import java.io.IOException;
+import nva.commons.utils.JsonUtils;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class EventTest {
+
+    public static final String SAMPLE_EVENT_JSON = "src/test/resources/sample_event.json";
```

Comment:
Use IoUtils.stringFromResources and you do not need to set such a path. This path is going to fail if someone runs the code on Windows.